server/repository/src/db_diesel/changelog/changelog.rs

```diff
@@ -117,8 +126,19 @@ impl<'a> ChangelogRepository<'a> {
         //     diesel::debug_query::<crate::DBType, _>(&query).to_string()
         // );
 
-        let result = query.load(&self.connection.connection)?;
-        Ok(result)
+        let result: Vec<ChangelogJoin> = query.load(&self.connection.connection)?;
+        Ok(result
+            .into_iter()
+            .map(|(change_log_row, name_link_row)| ChangelogRow {
+                cursor: change_log_row.cursor,
+                table_name: change_log_row.table_name,
+                record_id: change_log_row.record_id,
+                row_action: change_log_row.row_action,
+                name_id: name_link_row.map(|r| r.name_id),
+                store_id: change_log_row.store_id,
+                is_sync_update: change_log_row.is_sync_update,
+            })
```

Comment:
Nit: Probably will change later anyway when renaming the column... The ChangelogRow is not really the DB row anymore. Maybe have a private ChangelogRow DB struct (with name_link_id) and a new `pub struct Changelog`` (with name_id).